source/MaterialXGenMdl/MdlShaderGenerator.cpp

```diff
@@ -47,11 +47,12 @@ namespace
         "import ::state::*",
         "import ::anno::*",
         "import ::tex::*",
-        "import ::nvidia::core_definitions::*",
+        //"import ::nvidia::core_definitions::*",
         "import ::mx::stdlib::*",
         "import ::mx::swizzle::*",
         "import ::mx::supplib::*",
-        "import ::mx::pbrlib::*"
+        "import ::mx::pbrlib::*",
+        "using ::mx::core import *"
```

Comment:
@niklasharrysson as we discussed modifying to to use the "using" notation so we don't need to prefix anything in core with `mx::core::`